book/src/configuration-file.md

```diff
@@ -24,3 +28,20 @@ posix_fallocate = {}
 #### Feature configuration
 
 TODO
+
+#### file_flags
+
+Several UNIX operating systems have a concept of "file flags",
+which adds an additional level of security and control.
```

Comment:
Not all flags related to security, for example `arch`.

Reply:
Not sure about this one, I just took it from the FreeBSD's book definition.

Reply:
It was at "3.4.2. FreeBSD File Flags". Should I rephrase the sentence?

Reply:
Oh, that's because it's in a subsection on security, so security-related flags are the only thing that section cares about.  I think you can just remove the background bit.  How about "Some tests are related to file flags.  However, ..."

Reply:
Alright, thanks!